manifests/init.pp

```diff
@@ -101,6 +101,25 @@
 # [*token_retransmits_before_loss_const*]
 #   How many token retransmits before forming a new configuration
 #
+# [*join*]
+#   This timeout specifies in milliseconds how long to wait for join messages
+#   in the membership protocol.
+#   Default to 50
```

Comment:
Why was it set to 60 ?
By default in the man corosync.conf it's set to 50 milliseconds. That's why I just fit to the original values.